python/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+import os
+
+setup(
+    name='kfserving',
+    version='0.1.0',
+    author_email='ellisbigelow@google.com',
+    license='../../LICENSE.txt',
+    url='https://github.com/kubeflow/kfserving/model-servers/kfserver',
+    description='Model Server for arbitrary python ML frameworks.',
+    long_description=open('README.md').read(),
+    python_requires='>3.4',
```

Comment:
add these files under a versioned path so that support for other python versions can also be implemented?

Reply:
I was thinking we use git branching for this. WDYT?

I was also thinking each kfserver release version maps to a framework version tuple (f1v1, f2v2, f3v1).

Reply:
I prefer paths as that makes it easier to see what is supported and make changes... is using git branches an established pattern used elsewhere? Also if we were to go down the git branches route then this should be in a repo of its own.. branches where only part of the code changes and other code is stale seems eeky

Reply:
I would love to put this into a KFServing-core or KFServing-python repo.

Can we punt on this question for now? I want to enable an MVP. These sound like great feature requests and future work ;)